charts/rstudio-connect/README.md.gotmpl

```diff
@@ -64,6 +64,66 @@ Alternatively, database passwords may be set during `helm install` with the foll
 
 `--set config.Postgres.Password="<YOUR_PASSWORD_HERE>"`
 
+## Chronicle Agent
+
+This chart supports use of a sidecar Chronicle agent to report data to a Chronicle server. The agent can be enabled
```

Comment:
Should we link to the agent docs here?  We have a reference further down to the docs, but it may be good introduce earlier.
```suggestion
This chart supports use of a sidecar [Chronicle agent](https://docs.posit.co/chronicle/) to report data to a Chronicle server. The agent can be enabled
```

Reply:
I think that makes sense to include up here too